BookingsApi/BookingsApi.Infrastructure.Services/AsynchronousProcesses/ParticipantUpdateToHearingAsynchronousProcess.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using BookingsApi.Common.Services;
+using BookingsApi.Domain;
+using BookingsApi.Infrastructure.Services.Publishers;
+using System.Threading.Tasks;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Infrastructure.Services.IntegrationEvents.Events;
+
+namespace BookingsApi.Infrastructure.Services.AsynchronousProcesses
+{
+    public interface IParticipantUpdateToHearingAsynchronousProcess
+    {
+        Task Start(VideoHearing videoHearing);
+    }
+
+    public class ParticipantUpdateToHearingAsynchronousProcess : IParticipantUpdateToHearingAsynchronousProcess
```

Comment:
Can we rename this process, doesn't make grammatical sense

Reply:
I did

Reply:
sorry i must have missed it. I have changed the name now